ed_sensor_integration/src/laser/laser_plugin.h

```diff
@@ -49,27 +46,19 @@ class LaserPlugin : public ed::Plugin
 
     tf::TransformListener* tf_listener_;
 
-    geo::LaserRangeFinder lrf_model_;
+    LaserUpdater updater_;
 
     void scanCallback(const sensor_msgs::LaserScan::ConstPtr& msg);
 
+    /**
+     * @brief update update the worldmodel based on a novel laserscan message.
```

Comment:
Please check that there is no need for an empty line between `@biref` and the parameters. (Something that is needed in python docs generated by Sphinx)

Reply:
Where do I find what is needed by the documentation generation?